src/main/java/com/github/invictum/reportportal/ReportIntegrationConfig.java

```diff
@@ -80,6 +82,12 @@ public String communicationDirectory() {
         return System.getProperty(COMMUNICATION_DIR_KEY);
     }
 
+    public int retriesCount() {
+        String value = System.getProperty(FAILSAFE_RERUN_KEY);
+        value = value == null ? System.getProperty(SUREFIRE_RERUN_KEY) : value;
+        return value == null ? 0 : Integer.parseInt(value);
```

Comment:
`parseInt` may produce `NumberFormatException`, it is better to catch it and rethrow more informative exception in case of this event.

Reply:
It's unreal situation, because maven plugins will fail execution before:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-failsafe-plugin:3.0.0-M4:integration-test (default) on project ui-tests: Unable to parse configuration of mojo org.apache.maven.plugins:maven-failsafe-plugin:3.0.0-M4:integration-test for parameter rerunFailingTestsCount: Cannot convert 'lol' to int: For input string: "lol" -> [Help 1]
```
But I added catch `NumberFormatException` and throw more useful with explanations.